Public/Objects/Compare-ObjectProperties.ps1

```diff
@@ -4,16 +4,18 @@ Function Compare-ObjectProperties {
         [PSObject]$DifferenceObject,
         [switch]$CaseSensitive = $false
     )
-    $objprops = $ReferenceObject | Get-Member -MemberType Property, NoteProperty | ForEach-Object Name
-    $objprops += $DifferenceObject | Get-Member -MemberType Property, NoteProperty | ForEach-Object Name
+    $objprops = @(
+                  $($ReferenceObject | Get-Member -MemberType Property, NoteProperty | ForEach-Object Name),
+                  $($DifferenceObject | Get-Member -MemberType Property, NoteProperty | ForEach-Object Name)
+                  )
     $objprops = $objprops | Sort-Object | Select-Object -Unique
```

Comment:
This can be `Sort-Object -Unique`. No need for Select-Object.

Reply:
done